src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -1584,8 +1603,59 @@ export default class OncoprintControls extends React.Component<
                     <Observer>{this.getDownloadMenu}</Observer>
                     <Observer>{this.getHorzZoomControls}</Observer>
                     {this.minimapButton}
+                    <ConfirmNgchmModal ref={(ref) => this.confirmModal = ref} />
                 </ButtonGroup>
             </div>
         );
     }
 }
+
+
+export class ConfirmNgchmModal extends React.Component {
```

Comment:
this file is already huge, better to just put `ConfirmNgchmModal` in its own file

Reply:
@adamabeshouse Done with both of the requested changes.  Thanks!  Is everything else ok?